examples/heal_extended/component-restart/nsm-system-before/admission-webhook-k8s-patch.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: admission-webhook-k8s
+spec:
+  template:
+    spec:
+      containers:
+      - name: admission-webhook-k8s
+        env:
+        - name: NSM_ENVS
+          value: NSM_LOG_LEVEL=TRACE,NSM_LIVENESSCHECKENABLED=false,NSM_MAX_TOKEN_LIFETIME=2m,NSM_REQUEST_TIMEOUT=5s
```

Comment:
Why do we disable liveness check and change request timeout?

Reply:
I updated the PR.
Now we have two clients - one of them uses NSM_LIVENESSCHECKENABLED, the other doesn't. This covers more cases. Thanks!
NSM_MAX_TOKEN_LIFETIME and NSM_REQUEST_TIMEOUT are used to reduce test execution time.